mautic/mautic
# Sidebar logo bug

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | Y
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR contains code that removes `background-image` css property from `.mautic-brand` when the left sidebar is minimized.
It is causing UI issues for some instances.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Add this code to the .css of your Mautic instance:
```css

.mautic-brand svg {
  display: none;
}
.mautic-brand {
  background-image: url("https://www.mautic.org/sites/default/files/2020-03/Mautic_Logo_Monochrome_RGB_LB.png");
  background-position: center;
  background-repeat: no-repeat;
  background-size: 120px;
}
.mautic-logo svg {
  display: none;
}
.mautic-logo {
  background-image: url("https://www.mautic.org/sites/default/files/2020-03/Mautic_Logo_Monochrome_RGB_LB.png");
  background-position: center;
  background-repeat: no-repeat;
  background-size: 150px;
  height: 150px;
  border-radius: 0 !important;
}
.mautibot-image img {
  visibility: hidden;
}
.mautibot-image {
  background-image: url("https://www.mautic.org/sites/default/files/2020-03/Mautic_Logo_Monochrome_RGB_LB.png");
  background-repeat: no-repeat;
  border-radius: 0 !important;
  background-size: contain;
  background-position: center;
}

```
This css exactly matches what we have in one of our client instances.
2. Reload the page, minimize the left sidebar. You will see that the logo is cropped incorrectly.

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Repeat step 1 from "Steps to reproduce the bug".
3. Minimize the sidebar. You will see that background image is fitted inside the div (nicely resized to fit the div).
4. Maximize the sidebar. The background image(logo) should appear as normal.
